src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -3577,6 +3609,102 @@ namespace ReportSizingManager {
         }
     }
 
+    void setHeatingOAFracAndInletTempForZoneEq(Real64 &OutAirFrac, Real64 &CoilInTemp, Real64 &DesMassFlow)
```

Comment:
Anyone want to comment on these new functions? Too busy? Break out OutAirFrac, CoinInTemp and CoilinHumRat to individual functions? If change to individual functions change to Real64 instead of void ? etc.